src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -83,6 +83,8 @@ public static void UpdateSemantics(this UIView nativeView, IView view)
 
 			if (semantics == null)
 				return;
+			else
+				nativeView.IsAccessibilityElement = true;
```

Comment:
These should only get set if there's an actual value set for Description/Hint

Reply:
fixing now - but can you remind me why the null check is sufficient for Android and not iOS?

Reply:
That statement applies to both Android and iOS

Reply:
gotcha - will fix both here and in XCT soon